### DISCUSSION TITLE
Classmethod support?

### DISCUSSION THREAD

**Discussion by WillAyd** [wjakob/nanobind#416] [Q&A] [answered]
Is there any way to define a classmethod for an extension class? I see staticmethod support is well documented but couldn't find anything for classmethod

https://nanobind.readthedocs.io/en/latest/classes.html#frequently-used

**Comment by wjakob** [comment, marked as answer]
It's not yet supported, and I'm happy to accept external contributions to realize such a feature (with the usual caveats: the change will need to be sufficiently small/self-contained)
